tests/visual/label-angular.html

```diff
@@ -0,0 +1,89 @@
+<!doctype html>
+<html lang="en" class="k-typography k-no-animations">
+<head>
+    <title>Label (Angular)</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" data-role="kendo-theme" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <script src="assets/scripts.js"></script>
+
+    <style>
+        #test-area {
+            grid-template-columns: repeat(4, 1fr);
+        }
+    </style>
+</head>
+<body>
+
+    <div id="test-area" class="grid">
+
+        <span>Checkbox</span>
+        <span>Radio</span>
+        <span dir="rtl">Checkbox (RTL)</span>
+        <span dir="rtl">Radio (RTL)</span>
+
+        <section>
+            <input type="checkbox" class="k-checkbox" checked>
+            <kendo-label class="k-checkbox-label">
+                <label class="k-label">Checked</label>
+            </kendo-label>
```

Comment:
in this case, you have two `inline-bock` elements that will have a space between them. You are trying to compensate it by subtracting what you think might be the width of a space character. However, if those are placed inside a flex element, or otherwise there is no space between them, they will overlap.

We cannot handle this particular case, so we don't handle it.